one-acre-fund/ussd-sms
# Ser 225 agrodealer locator

Agrodealer locator

https://telerivet.com/p/PShop/service/SV567cd9b2c2081069/edit

testing on telerivet:

- dial in the short code for Rwanda pshop
- choose zero from the main menu to access the pshop's locator
- choose any of the districts (1,2,3). The option 4 is there in case the users don't find their district hence they can get an appropriate message;
- after choosing a district, there is a list of sectors in that district.
- choose a sector
- you should see a screen showing the location and the contact of the agrodealer
- one message should be sent to the farmer, and another to the credit officer.
